Resources/Prototypes/Body/Parts/reptilian.yml

```diff
@@ -0,0 +1,222 @@
+# TODO: Add descriptions (many)
+# TODO BODY: Part damage
+- type: entity
+  id: PartReptilian
+  parent: BaseItem
+  name: "reptilian body part"
+  abstract: true
+  components:
+  - type: Damageable
+    damageContainer: Biological
+
+- type: entity
+  id: TorsoReptilian
+  name: "reptilian torso"
+  parent: PartReptilian
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Mobs/Species/Reptilian/parts.rsi
+    state: "torso_m"
+  - type: Icon
+    sprite: Mobs/Species/Reptilian/parts.rsi
+    state: "torso_m"
+  - type: BodyPart
+    partType: Torso
+    size: 14
+    compatibility: Biological
+    mechanisms:
+    - OrganHumanHeart
+    - OrganHumanLungs
+    - OrganHumanStomach
+    - OrganHumanLiver
+    - OrganHumanKidneys
+#    criticalThreshold: 100
+#    deadThreshold: 150
+
+- type: entity
+  id: HeadReptilian
+  name: "reptilian head"
+  parent: PartReptilian
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Mobs/Species/Reptilian/parts.rsi
+    state: "head_m"
+  - type: Icon
+    sprite: Mobs/Species/Reptilian/parts.rsi
+    state: "head_m"
+  - type: BodyPart
+    partType: Head
+    size: 7
+    compatibility: Biological
+    vital: true
+    mechanisms:
+    - OrganHumanBrain
+    - OrganHumanEyes
+#    criticalThreshold: 50
+#    deadThreshold: 120
+  - type: Input
+    context: "ghost"
+  - type: DummyInputMover
+  - type: GhostOnMove
+
+- type: entity
+  id: LeftArmReptilian
+  name: "left reptilian arm"
+  parent: PartHuman
```

Comment:
Shouldn't this be parenting PartReptilian?

Reply:
checking